pkg/controller/acmechallenges/sync.go

```diff
@@ -131,6 +136,8 @@ func (c *controller) Sync(ctx context.Context, chOriginal *cmacme.Challenge) (er
 	if ch.Status.State == "" {
 		err := c.syncChallengeStatus(ctx, cl, ch)
 		if err != nil {
+			logf.V(logf.ErrorLevel).ErrorS(err, "error synchronizing with ACME server")
```

Comment:
It does not look like `ErrorS` is used anywhere else in the cert-manager code atm.

Reply:
Looks like `logf.V(logf.ErrorLevel).` is used a lot in these files, we should probably refactor this on a wider scale separate from this PR.

Reply:
hello, Tim @inteon . The issue is that the handleError function is not passed in a context from which to construct a decorated logger similarly to how the Logger on line 68 in sync.go is constructed. The decorated logger that renders the resource_name and resource_namespace on the log line helps triage issues greatly and it'd be good to see this used more. I also found that some log lines in the controller initialization that should be logged are not logged, potentially due to a race condition.